src/main/java/com/srans/nestserver/service/MailService.java

```diff
@@ -0,0 +1,79 @@
+package com.srans.nestserver.service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+import com.srans.nestserver.model.User;
+
+/**
+ * 
+ * @author Manish Kumar
+ *
+ */
+@Service
+public class MailService {
+
+	
+	private JavaMailSender javaMailSender;
+
+	/**
+	 * 
+	 * @param javaMailSender
+	 */
+	@Autowired
+	public MailService(JavaMailSender javaMailSender) {
+		this.javaMailSender = javaMailSender;
+	}
+
+	/**
+	 * This function is used to send mail without attachment.
+	 * 
+	 * @param user
+	 * @throws MailException
+	 */
+
+	public void sendEmail(User user) throws MailException {
+
```

Comment:
Man! I hope just copy pasted code, check what is our requirement, accordingly code, u need to send email to the tenant after registration, it means send email with all the info of tenant with nice template and use as part of the body itself , no attachment required, in your code I don't see from TenantControler you have called this service to send email with logic of sending tenant information which includes name, phone, email, etc along with booked hostel, floor, room, bed information.